public/js/user-info.js

```diff
@@ -14,6 +14,9 @@ $("#new-game-modal-button").click(function () {
 });
 
 $("#modal-close").click(function () {
+  document.getElementById("gameTitle").value = "";
+  document.getElementById("gameDescription").value = "";
+  document.getElementById("gameTrailer").value = "";
```

Comment:
Out of curiosity, why aren't you using jquery?

Reply:
$("#gameTitle").value = "";
I tried doing this, but it's not working

Reply:
So have a read of the jquery api. Just because you're doing it wrong, doesn't give you the right to do it differently :).

It's about learning. Choosing to do it the way you're comfortable with omits any progress.

Reply:
Thanks Strahinja!

I have corrected it!